Bingha/Bingha/Views/Stat/Cells/StatisticsCollectionViewCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  StatisticsCollectionViewCell.swift
+//  Bingha
+//
+//  Created by Terry Koo on 2022/07/27.
+//
+
+import UIKit
+
+class StatisticsCollectionViewCell: UICollectionViewCell {
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var cardView: UIView!
+    @IBOutlet weak var reducedCarbonLabel: UILabel!
+    @IBOutlet weak var walkingDistance: UILabel!
```

Comment:
UILabel타입 변수의 변수명 끝에 Label을 붙이면 UI변수와 데이터변수를 구분하는데 도움이 될 것 같습니다!
```suggestion
    @IBOutlet weak var walkingDistanceLabel: UILabel!
```